packages/compass-app-stores/src/stores/instance-store.ts

```diff
@@ -221,7 +221,7 @@ export function createInstancesStore(
       const db =
         instance.databases.get(database) ??
         // We might be adding collection to a new db namespace
-        instance.databases.add({ _id: database });
+        instance.databases.add({ _id: database, name: database });
```

Comment:
Drive-by: small bug here with not passing the `name`, it's the same value as `_id`, but compared to collection model is not a derived value on database model, so will be missing until we fetch the db below. Never popped up in compass as an issue, but in compass-web we're using this value to render the CTA for adding data to the cluster